srcpkgs/cherrytree/template

```diff
@@ -1,25 +1,22 @@
 # Template file for 'cherrytree'
 pkgname=cherrytree
-version=0.38.9
+version=0.99.21
 revision=1
-archs=noarch
-build_style=python2-module
-pycompile_dirs="usr/share/cherrytree/modules"
-hostmakedepends="gettext python-devel desktop-file-utils"
-depends="pygtksourceview python-dbus python-enchant desktop-file-utils"
+wrksrc=${pkgname}_${version}
+build_style=cmake
+hostmakedepends="gettext pkg-config desktop-file-utils
+python3 glib-devel"
+makedepends="cpputest uchardet-devel libcurl-devel sqlite-devel
+libxml++-devel gtksourceviewmm-devel gspell-devel gtkmm-devel"
```

Comment:
I think the convention is indent 1 spaces inside quote

Reply:
perhaps `cpputest` should go in `checkdepends`?

Reply:
No, cpputest must be placed in makedepends

Reply:
Because BUILD_TESTING defaulted to ON